client/src/main/java/com/vaadin/client/communication/MessageHandler.java

```diff
@@ -1688,6 +1697,16 @@ public String getCsrfToken() {
     }
 
     /**
+     * Gets the push connection identifier for this session. Used when
+     * establishing a push connection with the client.
+     *
+     * @return the push connection identifier string
```

Comment:
@since 8.1 was added for this and several other methods in the commit "Use separate identifier for push connections" by a separate commit that added a lot of missing @since tags - those need to be updated both in master and in 8.0 if we pick this commit.